src/condor_utils/file_transfer.cpp

```diff
@@ -235,12 +235,13 @@ FileTransfer::~FileTransfer()
 }
 
 int
-FileTransfer::SimpleInit(ClassAd *Ad, bool want_check_perms, bool is_server, 
+FileTransfer::SimpleInit(ClassAd *Ad, bool want_check_perms, bool is_server,
 						 ReliSock *sock_to_use, priv_state priv,
 						 bool use_file_catalog, bool is_spool) 
 {
 	char buf[ATTRLIST_MAX_EXPRESSION];
 	char *dynamic_buf = NULL;
+	const bool allow_inline_plugins = true; // enable job TransferPlugins attribute
```

Comment:
Please create an internal param in the param table for this (otherwise there's not much point to this as it simply controls invocation of a single line).

I believe this can be treated as a "hidden" param for now.